### ISSUE TITLE
Sparse tests failing on master

### ISSUE THREAD

**Issue by max-sixty** [pydata/xarray#3265]
https://dev.azure.com/xarray/xarray/_build/results?buildId=695

```python

=================================== FAILURES ===================================
_______________________ TestSparseVariable.test_unary_op _______________________

self = <xarray.tests.test_sparse.TestSparseVariable object at 0x7f24f0b21b70>

    def test_unary_op(self):
>       sparse.utils.assert_eq(-self.var.data, -self.data)
E       AttributeError: module 'sparse' has no attribute 'utils'

xarray/tests/test_sparse.py:285: AttributeError
___________________ TestSparseVariable.test_univariate_ufunc ___________________

self = <xarray.tests.test_sparse.TestSparseVariable object at 0x7f24ebc2bb38>

    def test_univariate_ufunc(self):
>       sparse.utils.assert_eq(np.sin(self.data), xu.sin(self.var).data)
E       AttributeError: module 'sparse' has no attribute 'utils'

xarray/tests/test_sparse.py:290: AttributeError
___________________ TestSparseVariable.test_bivariate_ufunc ____________________

self = <xarray.tests.test_sparse.TestSparseVariable object at 0x7f24f02a7e10>

    def test_bivariate_ufunc(self):
>       sparse.utils.assert_eq(np.maximum(self.data, 0), xu.maximum(self.var, 0).data)
E       AttributeError: module 'sparse' has no attribute 'utils'

xarray/tests/test_sparse.py:293: AttributeError
________________________ TestSparseVariable.test_pickle ________________________

self = <xarray.tests.test_sparse.TestSparseVariable object at 0x7f24f04f2c50>

    def test_pickle(self):
        v1 = self.var
        v2 = pickle.loads(pickle.dumps(v1))
>       sparse.utils.assert_eq(v1.data, v2.data)
E       AttributeError: module 'sparse' has no attribute 'utils'

xarray/tests/test_sparse.py:307: AttributeError
```

Any ideas?

**Comment by dcherian** [comment]
Doesn't seem like there's anything in the public API that we can use: https://sparse.pydata.org/en/latest/generated/sparse.html

**Comment by dcherian** [comment]
> It is probably also worth removing a bunch of "xfail" marks that are resolved by the sparse upgrade.

I can do this is #3255

**Comment by dcherian** [comment]
Closed by #3255
